dolphin/db/sqlalchemy/models.py

```diff
@@ -47,26 +47,27 @@ def to_dict(self):
 class AccessInfo(BASE, DolphinBase):
     """Represent access info required for storage accessing."""
     __tablename__ = "access_info"
-    storage_id = Column(String(128), primary_key=True)
-    hostname = Column(String(128))
-    username = Column(String(128))
-    password = Column(String(128))
+    storage_id = Column(String(36), primary_key=True)
```

Comment:
Trying to understand: Why is Storage_id the PK in access_info table? Should not this be FK or Composite? storage_id is ideally part of storage table